homedir.nix

```diff
@@ -3,6 +3,7 @@
     source = ./home;
   };
   system.activationScripts.script.text = ''
+    if [[ $(who | grep fablab) > 0 ]]; then echo logged in; fi
```

Comment:
is this echo from testing? I guess you want an exit 0?

Reply:
@araab You're right, I guess I copied the wrong statement. Fixed in b960f5a2